README.md

```diff
@@ -50,17 +50,14 @@ public class Main {
     private static final Logger LOGGER = Logger.getLogger(Main.class.getName());
 
     public static void main(String[] args) {
-        try {
+        try (SivaReader sivaReader = new SivaReader(new File(DEFAULT_SIVA_FILE))) {
```

Comment:
sorry @berry120 , could you get out of the `try` block the logging line `LOGGER.log(Level.INFO, "unpacking siva-file")` so this is logged although the siva reader opening fails?

Thanks for your contribution!

Reply:
Yup, another good point! That's now done too.